internal/dataplane/translator/translator_test.go

```diff
@@ -5060,12 +5102,17 @@ func (p fakeSchemaServiceProvier) GetSchemaService() kong.AbstractSchemaService
 }
 
 func mustNewTranslator(t *testing.T, storer store.Storer) *Translator {
-	p, err := NewTranslator(zapr.NewLogger(zap.NewNop()), storer, "", FeatureFlags{
-		// We'll assume these are true for all tests.
-		FillIDs:                           true,
-		ReportConfiguredKubernetesObjects: true,
-		KongServiceFacade:                 true,
-	}, fakeSchemaServiceProvier{},
+	logger := zapr.NewLogger(zap.NewNop())
+	clusterDomain := "cluster.local"
```

Comment:
same here

Reply:
👍 That would cause an import cycle so I needed to extract manager's consts like so dabe77fed657ea0781bea0afa9f770d44b9e88b2

PTAL